src/components/Toolbar/Toolbar.jsx

```diff
@@ -49,17 +49,17 @@ function Toolbar() {
           {/* Main links  */}
           <span className={classes[0]}>
             <li>
-              <NavLink to='/contact-us' activeClassName='active-link'>
+              <NavLink to='/contact-us' activeClassName='active-link' onClick={handleToggle}>
```

Comment:
if the screen is in desktop view and a link is clicked then it will toggle the nav open and this will be seen if the screen is made smaller. We only want to call handleToggle if the menu is open. This will mean it was toggled open while in mobile view